rt_dashboard/contrib/django/__init__.py

```diff
@@ -16,6 +17,7 @@ def get_admin_view(base_path):
 
     base_view = get_view(base_path + '/inner')
 
+    @csrf_exempt
```

Comment:
I think it would be safer to add the CSRF token to the request:
https://docs.djangoproject.com/en/3.2/ref/csrf/#ajax